config/admin-functions/reset-totp.sh

```diff
@@ -2,5 +2,5 @@
 
 set -euo pipefail
 
-sudo tpm2-totp clean || true
+sudo tpm2-totp clean 2>/dev/null || true
```

Comment:
Why?

Reply:
If totp hasn't been initialized, the `clean` command prints an error saying you need to initialize it first. The `|| true` here handles that case by ignoring the error exit code, but I also wanted to suppress the output.